utils/build-mlir-air-using-wheels.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+#set -x
+
+##===- utils/build-mlir-air-using-wheels.sh - Build mlir-air --*- Script -*-===##
+# 
+# Copyright (C) 2023, Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: MIT
+
+# 
+##===----------------------------------------------------------------------===##
+#
+# This script build mlir-air using wheels released from Xilinx/mlir-aie repository. 
+#
+# Note: released mlir-aie wheels require the system to have gcc version >= 11
+#       and Python 3.10 and above. For lit tests, please set -DLLVM_EXTERNAL_LIT
+#       path.
+#
+# <build dir>    - optional, build dir name, default is 'build'
+# <install dir>  - optional, install dir name, default is 'install'
+#
+##===----------------------------------------------------------------------===##
+
+BUILD_DIR=${1:-"build"}
+INSTALL_DIR=${2:-"install"}
+MLIR_WHL_DIR="mlir_wheel"
+MLIR_AIE_WHL_DIR="mlir_aie_wheel"
+
+mkdir -p $BUILD_DIR
+mkdir -p $INSTALL_DIR
+cd $BUILD_DIR
+mkdir -p $MLIR_WHL_DIR
+mkdir -p $MLIR_AIE_WHL_DIR
+
+MLIR_DISTRO="mlir_no_rtti-18.0.0.2023121521+d36b483-py3-none-manylinux_2_27_x86_64.manylinux_2_28_x86_64.whl"
+MLIR_AIE_DISTRO="mlir_aie_no_rtti-0.0.1.2023121602+5631ba5c-py3-none-manylinux_2_28_x86_64.whl"
```

Comment:
you can just do 

```
export PIP_FIND_LINKS="https://github.com/Xilinx/mlir-aie/releases/expanded_assets/mlir-distro https://github.com/Xilinx/mlir-aie/releases/expanded_assets/latest-wheels"
pip download mlir_no_rtti==18.0.0.2023121521+d36b483
pip download mlir_aie_no_rtti==0.0.1.2023121602+5631ba5c
```

which will work on all platforms (not just linux - ie you don't need the `manylinux_2_28_x86_64` at the end).

Reply:
Thanks for the pointer! Script is updated.